connections.go

```diff
@@ -75,21 +77,24 @@ type Connection struct {
 	Organization *Organization        `json:"organization,omitempty"`
 	Speed        int                  `json:"speed,omitempty"`
 	Token        string               `json:"token,omitempty"`
+	Tokens       []FabricServiceToken `json:"tokens,omitempty"`
 	Tags         []string             `json:"tags,omitempty"`
 	Ports        []ConnectionPort     `json:"ports,omitempty"`
 }
 
 type ConnectionCreateRequest struct {
-	Name        string               `json:"name,omitempty"`
-	Redundancy  ConnectionRedundancy `json:"redundancy,omitempty"`
-	Facility    string               `json:"facility,omitempty"`
-	Metro       string               `json:"metro,omitempty"`
-	Type        ConnectionType       `json:"type,omitempty"`
-	Mode        ConnectionMode       `json:"mode,omitempty"`
-	Description *string              `json:"description,omitempty"`
-	Project     string               `json:"project,omitempty"`
-	Speed       int                  `json:"speed,omitempty"`
-	Tags        []string             `json:"tags,omitempty"`
+	ContactEmail     string                 `json:"contact_email,omitempty"`
+	Description      *string                `json:"description,omitempty"`
+	Facility         string                 `json:"facility,omitempty"`
+	Metro            string                 `json:"metro,omitempty"`
+	Mode             ConnectionMode         `json:"mode,omitempty"`
+	Name             string                 `json:"name,omitempty"`
+	Project          string                 `json:"project,omitempty"`
+	Redundancy       ConnectionRedundancy   `json:"redundancy,omitempty"`
+	ServiceTokenType FabricServiceTokenType `json:"service_token_type,omitempty"`
+	Speed            int                    `json:"speed,omitempty"`
+	Tags             []string               `json:"tags,omitempty"`
+	Type             ConnectionType         `json:"type,omitempty"`
 }
```

Comment:
A new field that is being added is `vlans` `[]int{}` (vlan ids): A list of VLANs that will be attached on shared connections. The first VLAN defined will be used for the primary port, and the second VLAN will be assigned to the secondary port when `type` is redundant.